seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/JobMetricsServiceImpl.java

```diff
@@ -183,17 +183,17 @@ private Map<Long, JobSummaryMetricsRes> getMatricsListIfTaskTypeIsBatch(
                 } else {
                     log.info(
                             "The job does not exist on the engine, it is directly returned from the database");
-                    JobSummaryMetricsRes jobMetriceFromDb =
+                    JobSummaryMetricsRes jobMetricsFromDb =
                             getJobSummaryMetricsResByDb(
                                     jobInstance,
                                     userId,
                                     Long.toString(
                                             jobInstanceIdAndJobEngineIdMap.get(
                                                     jobInstance.getId())));
-                    if (jobMetriceFromDb != null) {
-                        jobSummaryMetricsResMap.put(jobInstance.getId(), jobMetriceFromDb);
+                    if (jobMetricsFromDb != null) {
+                        jobSummaryMetricsResMap.put(jobInstance.getId(), jobMetricsFromDb);
                     }
-                    if (jobInstance.getJobStatus().equals("RUNNING")) {
+                    if ("RUNNING".equals(jobInstance.getJobStatus())) {
```

Comment:
Now that you've fixed this problem, how about StringUtils.euqles here

Reply:
Of course, it's perfectly fine to leave it here

Reply:
I think the code is clean and easy-read, using StringUtils adds complexity.

Reply:
Yes, you don't have to change it here